includes/controller/user_angeltypes_controller.php

```diff
@@ -167,12 +177,74 @@ function user_angeltype_confirm_controller(): array
         throw_redirect(page_link_to('angeltypes', ['action' => 'view', 'angeltype_id' => $angeltype['id']]));
     }
 
+    user_angeltype_confirm_email($user_source, $angeltype);
+
     return [
         __('Confirm angeltype for user'),
         UserAngelType_confirm_view($user_angeltype, $user_source, $angeltype)
     ];
 }
 
+/**
+ * @param User  $user
+ * @param array $angeltype
+ * @return void
+ */
+function user_angeltype_confirm_email(User $user, array $angeltype): void
```

Comment:
The confirm email should also be sent, when the user is added to the angeltype by a buerocrat without applying for it before.

Reply:
There is an email?